### PR TITLE
Docs: Listener configuration, Authentication and Authroization

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Documentation

### Description

This PR adds documentation about listener configuration, authntication and authorization.

### Checklist

- [x] Update documentation

### CODE REVIEW

File: documentation/book/assembly-configuring-kafka-listeners.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+:parent-context: {context}
+
+[id='assembly-configuring-kafka-broker-listeners-{context}']
+= Configuring Kafka broker listeners
+
+{ProductName} allows users to configure the listeners which will be enabled in Kafka brokers.
+Two types of listeners are supported:
+
+* Plain listener on port 9092 (without encryption)
+* TLS interface on port 9093 (with encryption)
```
Comment (ppatierno) [inline documentation/book/assembly-configuring-kafka-listeners.adoc]: Why "interface" and not "listener" as the previous one?

Comment (scholzj) [reply documentation/book/assembly-configuring-kafka-listeners.adoc]: Should be probably listener. You are right. I will fix it.

File: documentation/book/ref-kafka-listeners.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring-kafka-listeners.adoc
+
+[id='ref-kafka-listeners-{context}']
+= Kafka listeners
+
+Kafka broker listeners can be configured using the `listeners` property in the `Kafka.spec.kafka` resource.
+The `listener` property contains two sub-properties:
```
Comment (dmuntima) [inline documentation/book/ref-kafka-listeners.adoc]: add "s" to "listener" > The `listeners` property

File: documentation/book/ref-kafka-listeners.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+[source,yaml,subs="attributes+"]
+----
+# ...
+listeners:
+  plain: {}
+# ...
+----
+
+The listener sub-properties might also contain additional configuration.
+Currently the only supported configuration is authentication on the `tls` listener.
```
Comment (dmuntima) [inline documentation/book/ref-kafka-listeners.adoc]: Add comma after "Currently".

File: documentation/book/ref-kafka-authentication.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+// assembly-kafka-authentication-and-authorization.adoc
+
+[id='ref-kafka-authentication-{context}']
+= Authentication
+
+Authentication is configured as part of the xref:assembly-configuring-kafka-broker-listeners-{context}[listener configuration] in the `authentication` property.
+When the `authentication` property is missing, no authentication will be enabled on given listener.
+The authentication mechanism which will be used is defined by the `type` field.
+
+Currently, the only support authentication mechanism is the TLS client authentication.
```
Comment (dmuntima) [inline documentation/book/ref-kafka-authentication.adoc]: support > supported

File: documentation/book/ref-kafka-authorization.adoc [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+Currently, the only supported authorization method is the Simple authorization.
+
+== Simple authorization
+
+Simple authorization is using the `SimpleAclAuthorizer` plugin.
+`SimpleAclAuthorizer` is the default authorization plugin which is part of Apache Kafka.
+To enable simple authorization, the `type` field should be set to `simple`.
+
+.An example of Simple auhtorization
```
Comment (dmuntima) [inline documentation/book/ref-kafka-authorization.adoc]: typo: auhtorization > authorization

**Comment by scholzj** [comment]
@dmuntima Thanks. All review comments should be incorporated.

**Pull request merged**